schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/ISchemaRegistryService.java

```diff
@@ -113,6 +113,13 @@
     SchemaIdVersion addSchemaVersion(String schemaName, SchemaVersion schemaVersion) throws InvalidSchemaException, IncompatibleSchemaException, SchemaNotFoundException;
 
     /**
+     *  Deletes a schema version given {@code schemaVersionKey}, throws an SchemaNotFoundException if the schema version is not absent.
```

Comment:
typo nit: Deletes a schema version for the given {@code schemaVersionKey}, throws SchemaNotFoundException if the schema version (is not ~~absent~~ present). I am also fine with removing throws here as it is already covered in throws javadoc.